modules/management.am

```diff
@@ -206,6 +206,35 @@ function _launcher(){
 	fi
 }
 
+function _launchers_not_found() {
+	MISSING_LAUNCHERS_MSG=" No launcher found."
+	printf "%b\n%b\n%b\n" "$DIVIDING_LINE" "$MISSING_LAUNCHERS_MSG" "$DIVIDING_LINE"
+}
+
+function _launcher_update() {
+	_clean_launchers 2>/dev/null 1>/dev/null
+	MISSING_LAUNCHERS_MSG="No launcher found, use option \"--launcher\" to create them."
+	[ ! -d "$DATADIR"/applications/AppImages ] && _launchers_not_found && exit 0
+	[ -d "$DATADIR"/applications/AppImages ] && [ -z "$( ls -A "$DATADIR"/applications/AppImages )" ] && _launchers_not_found && exit 0
+	if ! command -v appimageupdatetool 1>/dev/null; then
+		update_launchers_error_message=" 💀 ERROR! Missing command \"${RED}appimageupdatetool\033[0m\", install it and retry!"
+		printf "%b\n%b\n%b\n" "$DIVIDING_LINE" "$update_launchers_error_message" "$DIVIDING_LINE"
+		exit 0
+	else
+		echo "$DIVIDING_LINE"
+		for var in "$DATADIR"/applications/AppImages/*.desktop; do
+			appimage_full_path=$(grep "^Exec=" 0<"$var" 2>/dev/null | head -1 | cut -c 6- | sed 's/"//g; s/\s.*$//')
```

Comment:
this can just be `appimage_full_path="$(awk -F'=| ' '/Exec=/{print $2; exit}' "$var")"`

Which means the first `Exec=` we print "$2" by using `=` and `space` as separator, then we exit (head -1 equivalent here).

Reply:
I have this command for all three use cases: `--launcher`, `-c` and `-f`

if you want to take care about them (it is referenced only three times, one per script) you can PR me in dev

Reply:
Ok but I'm about to sleep, if you want I will later replace all 3 for the awk trick tomorrow.

Reply:
just tested your line, didn't worked
```
Failed to resolve full path to AppImage: No such file or directory
Could not read file:
```